File: Python/Estrutura condicional/temperatura.py
# Entrada de dados
escala = input("Escolha uma escala (C/F): ")

# Condicional
if escala == 'F' or escala == 'f':
    fahrenheit = float(input("Digite a temperatura em Fahrenheit: "))
    celsiustof = (5 / 9) * (fahrenheit - 32)
    print(f"A temperatura equivale a {celsiustof:.2f} ºF.")

elif escala == 'C' or escala == 'c':
    celsius = float(input("Digite a temperatura em Celsius: "))
    fahrenheittoc = celsius * 1.8 + 32
    print(f"A temperatura equivale a {fahrenheittoc:.2f} ºC.")

else:
    print("Escala inválida!")

File: C#/Estrutura condicional/temperatura.cs
// Variáveis
string escala;
double celsius, celsiustof, fahrenheit, fahrenheittoc;

// Entrada de dados
Console.Write("Escolha a escala (C/F): ");
escala = Console.ReadLine();

// Condicional
if (escala == "F")
{
    Console.Write("Digite a temperatura em Fahrenheit: ");
    fahrenheit = double.Parse(Console.ReadLine());
    celsiustof = (5.0 / 9.0) * (fahrenheit - 32);
    Console.Write("A temperatura equivale a " +  celsiustof.ToString("F2") + "ºC.");
}

else if (escala == "C")
{
    Console.Write("Digite a temperatura em Celsius: ");
    celsius = double.Parse(Console.ReadLine());
    fahrenheittoc = celsius * 1.8 + 32;
    Console.Write("A temperatura equivale a " + fahrenheittoc.ToString("F2") + "ºF.");
}

else
{
    Console.Write("A escala é inválida!");
}